apps/app/dashboards/healthcare/covid-vaccination/vaccine-trends.tsx

```diff
@@ -44,16 +65,30 @@ const COVIDVaccinationTrends: FunctionComponent<COVIDVaccinationTrendsProps> = (
             <Timeseries
               title={t("combine_title", {
                 state: CountryAndStates[currentState],
+                context: data.periodly,
               })}
-              interval="auto"
+              className="h-[300px]"
+              menu={
+                <Tabs.List
+                  options={[t("daily_7d"), t("daily"), t("monthly"), t("yearly")]}
```

Comment:
these `daily_7d`, `daily` etc is being repeated in many places. can you place these i18n inside the `common` namespace. So you dont have to repeat the same translation in other namespaces.

Reply:
Apply the same at other dashboards. Right now, the referred i18n for this page is broken